packages/expo-modules-core/android/src/main/java/expo/modules/core/errors/ModuleDestroyedException.kt

```diff
@@ -0,0 +1,22 @@
+package expo.modules.core.errors
+
+import kotlinx.coroutines.CancellationException
```

Comment:
To avoid adding Gradle dependency, I could instead import `java.util.concurrent.CancellationException`, because the Kotlin one is a typealias, but I think this solution is more stable.